sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeCustomFormsOperation.cs

```diff
@@ -60,17 +63,21 @@ public class RecognizeCustomFormsOperation : Operation<IReadOnlyList<RecognizedF
         /// </summary>
         /// <param name="operations"></param>
         /// <param name="diagnostics"></param>
-        /// <param name="modelId"></param>
         /// <param name="operationLocation"></param>
-        internal RecognizeCustomFormsOperation(ServiceClient operations, ClientDiagnostics diagnostics, string modelId, string operationLocation)
+        internal RecognizeCustomFormsOperation(ServiceClient operations, ClientDiagnostics diagnostics, string operationLocation)
         {
             _serviceClient = operations;
             _diagnostics = diagnostics;
-            _modelId = modelId;
 
             // TODO: Add validation here
             // https://github.com/Azure/azure-sdk-for-net/issues/10385
-            Id = operationLocation.Split('/').Last();
+
+            string[] substrs = operationLocation.Split('/');
+
+            _resultId = substrs[substrs.Length - 1];
```

Comment:
should we verify that we got the amount of partitioning we were expecting in the substrs so that we know that we have at least size 3?

Reply:
I considered doing that, but didn't bother since it's an `internal` constructor. What exception should we throw in this scenario? I think it wouldn't be relevant to the user since it would be an error on our side, not on theirs.

Reply:
We have plans for improving argument validation in the next previews, though. We may want to revisit this scenario.

Reply:
After discussing internally, we decided to throw an `ArgumentException` from the `public` constructor if parsing fails. The `internal` constructor will be left as it is because only a change from the service side could trigger an issue. We'll revisit it when discussing argument validation in the next previews.